pkg/store/database_test.go

```diff
@@ -7,204 +7,174 @@ import (
 	"github.com/nrc-no/core/pkg/api/types"
 	uuid "github.com/satori/go.uuid"
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/suite"
-	"gorm.io/driver/sqlite"
 	"gorm.io/gorm"
-	"testing"
 )
 
-type DatabaseSuite struct {
-	suite.Suite
-	db        *gorm.DB
-	dbFactory *factory
-	store     *databaseStore
-}
-
-func (d *DatabaseSuite) SetupSuite() {
-	db, err := gorm.Open(sqlite.Dialector{DSN: "file::memory:?cache=shared"}, &gorm.Config{
-		SkipDefaultTransaction: true,
-	})
-	if err != nil {
-		d.FailNow(err.Error())
-	}
-	d.db = db
-	if err := db.AutoMigrate(&Database{}, &Form{}, &Field{}); !assert.NoError(d.T(), err) {
-		d.FailNow(err.Error())
-	}
-	dbFactory := &factory{
-		db: db,
-	}
-	d.dbFactory = dbFactory
-
-}
-
-func (d *DatabaseSuite) SetupTest() {
-	s := &databaseStore{
-		createDatabase: func(db *gorm.DB, database *types.Database) error {
-			return nil
-		},
-		deleteDatabase: func(db *gorm.DB, databaseId string) error {
-			return nil
-		},
-		db: d.dbFactory,
-	}
-	d.store = s
-}
-
-func (d *DatabaseSuite) TestCreate() {
+// TestDatabaseCreate tests that we can create a database successfully
+func (d *Suite) TestDatabaseCreate() {
```

Comment:
some of these receivers are named `d`, while others are named `s`; could we choose one, just for the sake of consistency?

Reply:
Nice find, will update